test/integration/rollback_simulation_run_on_stop_and_destroy_sim_test.rb

```diff
@@ -0,0 +1,60 @@
+require 'test_helper'
+require 'db_helper'
+
+class RollbackSimulationRunOnStopAndDestroySimTest < ActionDispatch::IntegrationTest
+  include DBHelper
+
+  USER_NAME = 'user'
+  PASSWORD = 'password'
+  DUMMY = 'dummy'
+
+  def setup
+    super
+    user = ScalarmUser.new({login: USER_NAME})
+    user.password = PASSWORD
+    user.save
+    post login_path, username: USER_NAME, password: PASSWORD
+
+    simulation = Simulation.new(name: 'name', description: 'description',
+                                input_parameters: {}, input_specification: [])
+    simulation.user_id = user.id
+    simulation.save
+
+    experiment = Experiment.new({})
+    experiment.experiment_input = Experiment.prepare_experiment_input(simulation, [], [])
+    experiment.save
+    @@experiment_id = experiment.id
```

Comment:
Działa z instancyjnymi, muszę to w takim razie poprawić w kilku miejscach, musiałem coś źle zrobić wcześniej.